### PR TITLE
update overview.md

### PR DESCRIPTION
**Pull request by brightredchilli**
Update to assign some people to tasks.

### CODE REVIEW

File: ios/viewmodels/Overview.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ When in doubt, remember that the goal of introducing view models is to separate
 Topics/Proposed Topics:
 - [ ] Motivation behind using view models
 - [x] [View model reuse accross multiple models](view-model-reuse.md)
-- [ ] Should view models be value types as opposed to reference types?
-- [ ] View models with two way bindings(using rx)
+- [ ] Example of view model tests (Bryan)
+- [ ] View models with two way bindings using rx (Ben)
 - [ ] View models that handle network calls
 - [ ] View models that handle both network calls and persistence
```
Comment (paulrolfe) [inline ios/viewmodels/Overview.md:12]: I thought we just needed one of these network call topics in here, from our conversation? Are both necessary?

**Comment by paulrolfe** [comment]
:shipit: Just one question. ☝️

File: ios/viewmodels/Overview.md (hunk 1 above)
Comment (brightredchilli) [reply ios/viewmodels/Overview.md:12]: Given that having both lowers the cost of writing either of the posts, I would advocate for separating them.

Comment (paulrolfe) [reply ios/viewmodels/Overview.md:12]: 👍

**Review by Benuuu — APPROVED** [review]

**Pull request merged**